ui/app/src/views/home/ProjectsAndDashboards.tsx

```diff
@@ -99,17 +110,18 @@ function ProjectAccordion({ row }: ProjectAccordionProps) {
           />
         </AccordionDetails>
       </Accordion>
-      <DeleteProjectDialog
-        name={row.project}
-        open={openDeleteProjectDialog}
+      <DeleteResourceDialog
+        resource={row.project}
+        open={isDeleteProjectDialogOpen}
+        onSubmit={() => handleProjectDelete(row.project)}
         onClose={closeDeleteProjectConfirmDialog}
       />
     </>
   );
 }
 
 interface RenderDashboardListProps {
-  projectRows: ProjectRow[];
+  projectRows: ProjectDashboardsGroup[];
```

Comment:
I don't see the link of that part of the changes, with the project display name / rename

Reply:
Okay I understand now that you needed an additional get project query and an additional information display name in the data model. 

I'm ok then on the way it's been done since it probably gave you some headaches to generate that.